### ISSUE TITLE
The discrepancy of the user's guide

### ISSUE THREAD

**Issue by sergeyklay** [almostearthling/when-command#49]
> The following commands are needed for the program and the UI files:

> $ for x in share/*.glade ; do intltool-extract --type=gettext/glade $x ; done
> $ xgettext -k_ -kN_ -o po/messages.pot -D . -D share -f po/messages.list

But `po/messages.list` does not exist.

> To create a translation, one should be in a localized environment:

> export LANG=it_IT
> msginit --locale=it_IT

```
$ msginit --locale=ru_RU
msginit: Found no .pot file in the current directory.
         Please specify the input .pot file through the --input option.
Try 'msginit --help' for more information.
```

**Comment by almostearthling** [comment]
You're right, I actually changed its name to avoid confusion with `messages.pot`: its name is now `translate.list`. I'll update the `po/USING.md` file immediately, thank you for spotting this. And btw thank you for working on the translation! :clap:

**Comment by sergeyklay** [comment]
Also if it possible, it would be great if you able to create a small guide for [Poedit](https://poedit.net/).

**Comment by almostearthling** [comment]
Well, I'll try and see: I didn't use `poedit` -- the text was so short that I didn't take the time to install a new tool and just relied on the Atom editor. However I'll check it out, it might be still useful for updates.

**Comment by almostearthling** [comment]
Hi @sergeyklay, I just installed *poedit*, and it looks pretty user-friendly: what would you suggest as hints to provide to *poedit* users in **When**'s case?

**Comment by sergeyklay** [comment]
I think it will be enough to tell which file to use, how to use a common translation memory, what mistakes should not be allowed. Just a few strings :) I think in any case it would be good

**Comment by almostearthling** [comment]
Great, will do! :+:

**Comment by almostearthling** [comment]
I'd close this one, which has been fixed with the latest additions, and leave #52 open.

**Comment by sergeyklay** [comment]
:+1:
